jwql/website/apps/jwql/templates/api_landing.html

```diff
@@ -0,0 +1,152 @@
+{% extends "base.html" %}
+
+{% block preamble %}
+
+	<title>API - JWQL</title>
+
+{% endblock %}
+
+{% block content %}
+
+    <main role="main" class="container">
+        <h2>Explore the JWQL API</h2><hr>
+        <p align="justify">
+            <ul>
+                <li><a href="#introduction">What is the JWQL API?</a></li>
```

Comment:
I'm not sure why, but these seem to link to _just below_ where the respective section is, which is a bit annoying.  But if it is not an easy fix, it is probably not worth worrying about.

Reply:
Hmm, it seems that all links in general do this. I will look into it.

Reply:
Hey @bourque, everything I am seeing online links the `div id` by using `href=#div id` and I am not seeing any other solutions online. In order to save time and because this is the last conversation I need to resolve for this PR, I was thinking we can look into this later, maybe?

Reply:
Yeah sounds good.  It's just a minor annoyance, so not a big deal.